PENDING.md

```diff
@@ -61,6 +61,7 @@ CLI flag.
 
 ### SDK
 
+* \#3750 Track outstanding rewards per-validator instead of globally
```

Comment:
We should add a more comprehensive description of the fix (aka something like what you said "fixed the main problem, which was that slashes of re-delegations to a validator were not correctly accounted for in fee distribution when the redelegation in question had itself been slashed (from a fault committed by a different validator) in the same BeginBlock.")

Reply:
haha yes

Reply:
updated.